src/frontend/Sider/Oppgavebenk/filter/SaksbehandlerVelger.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { Select } from '@navikt/ds-react';
+
+import { useApp } from '../../../context/AppContext';
+import { OppgaveRequest } from '../typer/oppgave';
+
+interface Props {
+    oppgaveRequest: OppgaveRequest;
+    settOppgaveRequest: React.Dispatch<React.SetStateAction<OppgaveRequest>>;
+}
+const utledSaksbehandlerTekst = ({ tildeltRessurs, tilordnetRessurs }: OppgaveRequest) => {
+    if (tildeltRessurs === undefined && tilordnetRessurs === undefined) {
+        return 'Alle';
+    } else if (tilordnetRessurs) {
+        return tilordnetRessurs;
+    } else if (tildeltRessurs) {
+        return 'Fordelte';
+    } else {
+        return 'Ufordelte';
+    }
+};
+
+const SaksbehandlerVelger: React.FC<Props> = ({ oppgaveRequest, settOppgaveRequest }) => {
+    const { saksbehandler } = useApp();
+
+    const saksbehandlerTekst = utledSaksbehandlerTekst(oppgaveRequest);
+
+    return (
+        <Select
+            value={saksbehandlerTekst}
+            label="Saksbehandler"
+            onChange={(event) => {
+                const val = event.target.value;
+                if (val === '') {
+                    settOppgaveRequest((prevState: OppgaveRequest) => {
+                        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                        const { tildeltRessurs, tilordnetRessurs, ...rest } = prevState;
+                        return rest;
+                    });
+                } else if (val === 'Fordelte' || val === 'Ufordelte') {
+                    settOppgaveRequest((prevState: OppgaveRequest) => {
+                        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                        const { tildeltRessurs, tilordnetRessurs, ...rest } = prevState;
+                        return {
+                            ...rest,
+                            tildeltRessurs: val === 'Fordelte',
+                        };
+                    });
+                } else {
+                    settOppgaveRequest((prevState: OppgaveRequest) => {
+                        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                        const { tildeltRessurs, tilordnetRessurs, ...rest } = prevState;
+                        return {
+                            ...rest,
+                            tilordnetRessurs: val,
+                        };
+                    });
+                }
+            }}
+        >
+            <option value="">Alle</option>
```

Comment:
Jeg blir litt forvirret her. Hvorfor har Alle verdien `''`, men `utledSaksbehandlerTekst` vil returnere verdien `Alle` der tilordnetRessurs ikke er satt? Kunne ikke alle bare ha verdien `Alle` som dette?
```suggestion
            <option value="">Alle</option>
```

Det funker sikkert uten også, synes bare det var litt merkelig

Reply:
Man setter verdien til tom hvis man velger alle, blir det samme som å resette den

Reply:
I dette tilfelle går det, endret i onChange og